targets/ipq806x

```diff
@@ -1,7 +1,29 @@
-ATH10K_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+# The QCA9980 is discontinued from Qualcomm. It didn't receive any firmware-update since >4 years.
```

Comment:
"The QCA9980 **was** discontinued **by** Qualcomm. It didn't receive any firmware update **for** over 4 years."

Also add the link to the firmware repository for reference. 
https://github.com/kvalo/ath10k-firmware/tree/master/QCA99X0/hw2.0

Reply:
@blocktrron you missed to change this with your recent update

Reply:
fixed